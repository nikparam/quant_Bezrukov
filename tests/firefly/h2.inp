 $CONTRL SCFTYP=RHF RUNTYP=energy mult=1 units=bohr NPRINT=3 $END
 $BASIS GBASIS=cc-pvdz $END

 $DATA
 H2 scf test
C1
H 1.0 0.0 0.0 0.7 
H 1.0 0.0 0.0 -0.7
 $END
