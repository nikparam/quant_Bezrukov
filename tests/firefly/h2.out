                     Running on host artfin-MacBookPro

          ******************************************************
          *      Firefly version 8.2.0, build number 10203     *
          *   Compiled on    Monday,    19-09-2016, 15:30:23   *
          *Code development and Intel/AMD specific optimization*
          *  Copyright (c) 1994, 2016 by  Alex A. Granovsky,   *
          *          Firefly Project, Moscow, Russia.          *
          *   Some parts of this program include code due to   *
          * work of Jim Kress, Peter Burger, and Robert Ponec. *
          ******************************************************
          *             Firefly Project homepage:              *
          * http://classic.chem.msu.su/gran/firefly/index.html *
          *                      e-mail:                       *
          *               gran@classic.chem.msu.su             *
          *This program is not a free software and is provided *
          *exclusively to its registered users under the terms *
          *       of Firefly package license agreement         *
          * Unauthorized use of Firefly is strongly prohibited *
          *   This program may not be redistributed without    *
          * the specific, written permission of its developers.*
          ******************************************************

          ******************************************************
          *  PARTIALLY BASED ON  US GAMESS VERSION 6 JUN 1999, *
          *   US GAMESS VERSIONS  6 SEP 2001 AND 12 DEC 2003   *
          *             FROM IOWA STATE UNIVERSITY             *
          * M.W.SCHMIDT, K.K.BALDRIDGE, J.A.BOATZ, S.T.ELBERT, *
          *   M.S.GORDON, J.H.JENSEN, S.KOSEKI, N.MATSUNAGA,   *
          *          K.A.NGUYEN, S.J.SU, T.L.WINDUS,           *
          *       TOGETHER WITH M.DUPUIS, J.A.MONTGOMERY       *
          *         J.COMPUT.CHEM.  14, 1347-1363(1993)        *
          ******************************************************


 Core i7    / Linux  Firefly version running under Linux.
 Running on Intel CPU:  Brand ID  0, Family  6, Model  42, Stepping  7
 CPU Brand String    :  Intel(R) Core(TM) i5-2415M CPU @ 2.30GHz        
 CPU Features        :  x87 FPU, CMOV, MMX, SSE, SSE2, SSE3, SSSE3, SSE4.1, SSE4.2, AVX, HTT, MWAIT, EM64T
 Data cache size     :  L1 32 KB, L2  256 KB, L3  3072 KB
 max    # of   cores/package :   8
 max    # of threads/package :  16
 max     cache sharing level :  16
 actual # of   cores/package :   2
 actual # of threads/package :   4
 actual # of threads/core    :   2
 Operating System successfully passed SSE support test.
 Operating System supports AVX/FMA.


 PARALLEL VERSION (MPICH) RUNNING IN SERIAL MODE USING SINGLE PROCESS


 WARNING! THIS VERSION OF FIREFLY IS PROBABLY OUTDATED!
 PLEASE CHECK FIREFLY HOMEPAGE FOR INFORMATION ON UPDATES!

 EXECUTION OF FIREFLY BEGUN 12:31:23 21-APR-2018    

            ECHO OF THE FIRST FEW INPUT CARDS -
 INPUT CARD> $CONTRL SCFTYP=RHF RUNTYP=energy mult=1 units=bohr NPRINT=3 $END               
 INPUT CARD> $BASIS GBASIS=cc-pvdz $END                                                     
 INPUT CARD>                                                                                
 INPUT CARD> $DATA                                                                          
 INPUT CARD> H2 scf test                                                                    
 INPUT CARD>C1                                                                              
 INPUT CARD>H 1.0 0.0 0.0 0.7                                                               
 INPUT CARD>H 1.0 0.0 0.0 -0.7                                                              
 INPUT CARD> $END                                                                           
   20000000 WORDS OF MEMORY AVAILABLE

 This job is executing on     1 unique host(s)
 Minimum number of processes per host is:    1
 Maximum number of processes per host is:    1

 On master's host, detected    4 CPU core(s) in aggregate

 Warning: running without fastdiag runtime extension!

 Warning: HTT is enabled, bitmask of physically unique cores is 0x0000000C

 SMT aware parts of program will use              2 threads.

 Creating thread pool to serve up to            128 threads.

 Activating Call64 option.

 Using 64-bit DGEMM by default.

     BASIS OPTIONS
     -------------
     GBASIS=CC-PVDZ      IGAUSS=       0      POLAR=NONE    
     NDFUNC=       0     NFFUNC=       0     DIFFSP=       F
     NPFUNC=       0      DIFFS=       F


     RUN TITLE
     ---------
  H2 scf test                                                                    

 THE POINT GROUP OF THE MOLECULE IS C1      
 THE ORDER OF THE PRINCIPAL AXIS IS     0
 *** ERROR, ILLEGAL BASIS FUNCTION TYPE=CC-PVDZ  IGAUSS=   0
    LUCY: DO NOT WORRY, CHARLIE BROWN, WE LEARN MORE
          FROM LOSING THAN WE DO FROM WINNING.
 CHARLIE: THEN THAT MAKES ME THE SMARTEST PERSON IN
          THE WORLD!

          ILLEGAL          BASIS FUNCTION REQUESTED.

 ADDRESS 0x0919D7B5 HAS INITIATED PROGRAM ABORT BECAUSE OF FATAL ERROR(S)

 CPU        TIME:   STEP =      0.06 ,  TOTAL =        0.2 SECONDS (    0.0 MIN)
 WALL CLOCK TIME:   STEP =      0.06 ,  TOTAL =        0.1 SECONDS (    0.0 MIN)
 CPU UTILIZATION:   STEP =    106.11%,  TOTAL =     282.96%

      396840 WORDS OF    DYNAMIC MEMORY USED
      105664 BYTES OF    HEAP MEMORY    USED,       40640 BYTES REMAIN IN USE

 WARNING! THIS VERSION OF FIREFLY IS PROBABLY OUTDATED!
 PLEASE CHECK FIREFLY HOMEPAGE FOR INFORMATION ON UPDATES!

 EXECUTION OF FIREFLY TERMINATED ABNORMALLY AT 12:31:23 21-APR-2018    
